alacritty/src/main.rs

```diff
@@ -130,9 +130,14 @@ fn run(
 ) -> Result<(), Box<dyn Error>> {
     info!("Welcome to Alacritty");
 
-    info!("Configuration files loaded from:");
-    for path in &config.ui_config.config_paths {
-        info!("  \"{}\"", path.display());
+    // Log the configuration paths.
+    {
```

Comment:
What is this scope for?

Reply:
To drop things right away and don't keep them for the entire left time of the program.

Reply:
Should we maybe move this into a method then? If we need separation, might as well do it completely?